code/game/objects/items.dm

```diff
@@ -715,12 +715,13 @@ GLOBAL_DATUM_INIT(welding_sparks, /mutable_appearance, mutable_appearance('icons
 /obj/item/MouseEntered(location, control, params)
 	. = ..()
 	if(in_inventory || in_storage)
-		var/timedelay = 8
 		var/mob/user = usr
-		tip_timer = addtimer(CALLBACK(src, .proc/openTip, location, control, params, user), timedelay, TIMER_STOPPABLE)
+		var/mob/living/L = user
```

Comment:
Why do you move this line up here? It would be better to put it directly above line 727

Reply:
uhh, stupid habit from another language i was taught in school- i'll change it